docker/README.md

```diff
@@ -85,7 +87,7 @@ If you haven't already, clone the MMS repo and go into the `docker` folder.
 git clone https://github.com/awslabs/mxnet-model-server.git && cd mxnet-model-server/docker
 ```
 
-### Building the Docker Image
+### Building the Container Image
```

Comment:
I don't think we need capitalization here, so: "container image"

Reply:
Done